cfy_manager/status_reporter/status_reporter.py

```diff
@@ -123,3 +143,68 @@ def remove():
     logger.notice('Removing component status reporting service...')
     systemd.remove(STATUS_REPORTER)
     logger.notice('Component status reporting service removed')
+
+
+@argh.arg('--json',
+          help='Print the tokens in a JSON format instead of using logs.',
+          dest='json_',
+          default=False)
+def get_tokens(json_=None):
+    """Retrieves the status reporters' tokens from the DB and prints them to
+    STDOUT.
+
+    This must be run from a Manager machine.
+    """
+    logger.notice('Trying to fetch status reporters tokens...')
+    tokens = _get_status_reporter_tokens()
+    if not tokens:
+        logger.error("Failed to fetch tokens.")
+        return
+    if json_:
+        print(json.dumps(tokens))
+        return
+    for username, token in tokens.items():
+        logger.notice('Token of "{0}" is "{1}"'.format(username, token))
+
+
+def _get_status_reporter_tokens():
+    if not is_premium_installed():
+        logger.error("This command must be run on a Manager.")
```

Comment:
This command must be run on a Manager => This command can only be run on a Cloudify Manager machine